sympy/sets/fancysets.py

```diff
@@ -298,6 +298,16 @@ def TransformationSet(*args, **kwargs):
     return ImageSet(*args, **kwargs)
 
 
+class Complex(with_metaclass(Singleton, ImageSet)):
```

Comment:
Would it be possible to think of a name more suitable for a *set* (cf. "Reals"). (The name "Complex" might have other uses in the future.)